src/plugin.js

```diff
@@ -1,69 +1,80 @@
 'use strict';
 var Joi = require('joi');
+var _ = require('lodash');
+var otSchemas = require('ot-schemas');
+var SCHEMAS = _.get(otSchemas, 'searchApi.helpers.request', {});
 
-exports.register = function(plugin, options, next){
+var camelCaseSchema = _.mapKeys(SCHEMAS.COMBINED_SCHEMA, _.rearg(_.camelCase, 1));
+var keyMapping = _.mapKeys(_.mapValues(camelCaseSchema, function(value, key) { return key; }), function(value) { return _.toLower(value);});
+
+function endPreHandler(reply) {
+  if (_.isFunction(reply.continue)) {
+    return reply.continue();
+  }
+  return reply.continue;
+}
+
+exports.register = function(plugin, options) {
```

Comment:
is there a reason why you changed the above exports to module.exports but this one you kept?

Reply:
no i was modeling off of another module and just copy pasted. I can make them both module.exports.